code/modules/mob/living/carbon/human/human.dm

```diff
@@ -270,38 +270,50 @@
 				if(href_list["add_citation"])
 					var/max_fine = CONFIG_GET(number/maxfine)
 					var/citation_name = tgui_input_text(human_user, "Citation crime", "Security HUD")
+					var/citation_details = tgui_input_text(human_user, "Citation details", "Security HUD")
 					var/fine = tgui_input_number(human_user, "Citation fine", "Security HUD", 50, max_fine, 5)
 					if(!fine || !target_record || !citation_name || !allowed_access || !isnum(fine) || fine > max_fine || fine <= 0 || !human_user.canUseHUD() || !HAS_TRAIT(human_user, TRAIT_SECURITY_HUD))
+						playsound(human_user, 'sound/machines/terminal_prompt_deny.ogg', 30, FALSE, -3)
```

Comment:
I think you should use `SILENCED_SOUND_EXTRARANGE` (-11) instead of -3 here. It has no reason to be easily heard by others when there's no visual or chat feedback to back it up. Same with the other sounds from the HUD.